develop/app/Anser/Services/v2/ProductService.php

```diff
@@ -197,4 +197,112 @@ function (
             );
         return $action;
     }
+
+    /**
+     * Add inventory
+     *
+     * @param integer|null $p_key
+     * @param integer|null $addAmount
+     * @return ActionInterface
+     */
+    public function addInventory(?int $p_key = null, ?int $addAmount = null): ActionInterface
```

Comment:
I don't think the `$p_key` and `addAmount` parameters are optional, there isn't a scenario is call this function and don't need to pass these two parameters.

Reply:
This change is done.